### DISCUSSION TITLE
Best way to invest funds in the development of SageMath Inc., SageMath, and CoCalc

### DISCUSSION THREAD

**Discussion by nikkujensen** [sagemathinc/cocalc#6048] [General]
I have recently been blessed by the efforts of SageMath and CoCalc. 

I am wondering what are the best ways to support these endeavors such that they can thrive long term. It would be great if we could get a discussion going on this so it is clear.

**Comment by williamstein** [comment]
Hi, do you want to video chat sometime?  I'm the person who originally started bother SageMath and CoCalc and I'm happy to chat.  Feel free to email me at wstein@gmail.com.

**Comment by williamstein** [comment]
For SageMath donations, there's opencollective:  https://opencollective.com/sage_math

We have used funds from that so far for:

- this prize: https://www.sagemath.org/development-prize.html
- hosting of https://trac.sagemath.org/

For CoCalc, it's a small startup company...
